src/IO/VTK.cpp

```diff
@@ -722,7 +731,7 @@ void VTK::writeData( ){
                 }
 
                 str << std::endl ;
-                genericIO::flushBINARY( str, buffer, length) ;
+                genericIO::flushBINARY( str, buffer, length) ; // this is basically a reinterpret_cast of chars into chars. It should be safe in this context
```

Comment:
Can we remove this comments and the ones below? The previous comment is already stating why we are using a binary write for an ASCII file.

Reply:
done